tests/route_testing_conf.py

```diff
@@ -0,0 +1,7 @@
+"""
+A very basic python configuration file.
+Our single source of truth for which
+routes need to be tested and their expected
+content.
+"""
+routes_and_test_content = {"/": b"Hello World"}
```

Comment:
In the future we can automatically gather the routes :)

Reply:
Added as Todo in ticket in Jira FS-525